### DISCUSSION TITLE
请问目前功能中是否支持下载一个播放页面中多个播放列表的视频？

### DISCUSSION THREAD

**Discussion by qinfengluoye** [the1812/Bilibili-Evolved#3850] [提问 / Q&A]
脚本版本号：v2.5.2-17-g89d37a1f2
B站播放器：3.20.11
遇到问题：想下载的视频被官方号引流，点击就会进入一个毫不相干的节目页面中，此时无法提取到正确的下载列表。
具体情况：播放页面右侧有上下六七个播放列表，正片部分放的是官方节目，想下载的在最下播放列表中，但播放该视频并选择下载，出来的输入源仅有正片那个列表，不能显示切换其他列表，尝试很多方法没有成功，包括视频页面退回旧版。是操作不当还是功能不支持？

**Comment by Tinhone** [comment]
可以试一试安装插件：
```
下载视频 - 手动输入
downloadVideo.inputs.manual
```

**Comment by the1812** [comment]
当前没有直接支持, 可以用手动输入
